apps/aecore/test/aec_blocks_tests.erl

```diff
@@ -85,14 +85,21 @@ network_serialization_test_() ->
 
 validate_test_() ->
     {setup,
-     fun aec_test_utils:aec_keys_setup/0,
-     fun aec_test_utils:aec_keys_cleanup/1,
+     fun() ->
+             ok = meck:new(aec_chain, [passthrough]),
+             meck:expect(aec_chain, get_top_state, 0, {ok, aec_trees:new()}),
+             aec_test_utils:aec_keys_setup()
+     end,
+     fun(TmpKeysDir) ->
+             meck:unload(aec_chain),
+             ok = aec_test_utils:aec_keys_cleanup(TmpKeysDir)
+     end,
      [ {"Multiple coinbase txs in the block",
         fun validate_test_multiple_coinbase/0}
      , {"Malformed txs merkle tree hash",
         fun validate_test_malformed_txs_root_hash/0}
-     , {"Malformed tx signature",
-        fun validate_test_malformed_tx_signature/0}
+     %, {"Malformed tx signature",
```

Comment:
Remove the code instead of commenting.

Reply:
Yes, I am adding a corresponding test to verify this is covered by aec_trees. This comment will remove this test as well